java-checks/src/main/java/org/sonar/java/checks/SerializableFieldInSerializableClassCheck.java

```diff
@@ -41,7 +42,10 @@
 
 import javax.annotation.Nullable;
 
+import java.io.Serializable;
+import java.util.Collection;
 import java.util.List;
+import java.util.Map;
```

Comment:
As mentioned in other comment, those imports should not be required just to resolve the name of the class and thus be removed.